share/spice/quote_of_the_day/quote_of_the_day.css

```diff
@@ -1,55 +1,55 @@
-.zci-quote_of_the_day--container {
+.quote_of_the_day_container {
```

Comment:
These should really be namespaced with `.zci--quote_of_the_day .quote_of_the_day_container` -- if anything I would simplify the other class names: e.g. `qotd__container`